stl/inc/format

```diff
@@ -2557,8 +2557,221 @@ template <_Format_supported_charT _CharT, class _Traits>
 struct formatter<basic_string_view<_CharT, _Traits>, _CharT>
     : _Formatter_base<basic_string_view<_CharT, _Traits>, _CharT, _Basic_format_arg_type::_String_type> {};
 
-using format_context  = basic_format_context<back_insert_iterator<string>, string::value_type>;
-using wformat_context = basic_format_context<back_insert_iterator<wstring>, wstring::value_type>;
+template <class _T>

```

Comment:
The STL avoids macro like names with single letters. Conventionally we use `_Ty`

Reply:
Even worse, the Windows headers have a macro named `_T`. It will break this.

Reply:
It won't break because that one is a function-style macro but I fixed it.